test/assert-coverage-test.js

```diff
@@ -0,0 +1,24 @@
+const {describe, it} = require('mocha');
+const {getCoverageResults} = require('./coverage-utils');
+const expect = require('expect');
+
+describe('API coverage test', () => {
+  it('calls every method', () => {
+    if (!process.env.COVERAGE) return;
+
+    const coverageMap = getCoverageResults();
+    const missingMethods = [];
+    for (const method of coverageMap.keys()) {
+      if (!coverageMap.get(method))
+        missingMethods.push(method);
+    }
+    if (missingMethods.length) {
+      console.error('\nCoverage check failed: not all API methods called. See above output for list of missing methods.');
+      console.error(Array.from(missingMethods).join('\n'));
```

Comment:
I don't understand why we need `Array.from` here. How about this:

```suggestion
      console.error(missingMethods.join('\n'));
```

lmk if I'm missing something

Reply:
No nor me - this was left over from pre-me working on it. Nice spot!